examples/test_examples.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pathlib
+import pytest
+import subprocess
+from ruamel import yaml
+
+from determined import experimental, load
```

Comment:
blocking: it seems that these are not used

we probably want to add flake8 in for linting the tests